packages/core/content-releases/package.json

```diff
@@ -64,11 +64,11 @@
     "@strapi/design-system": "1.12.0",
     "@strapi/helper-plugin": "4.14.4",
     "@strapi/icons": "1.12.0",
+    "@strapi/strapi": "4.14.4",
```

Comment:
you actually want this to be a devDep & a peer dep, because this will be in `core` you'll create a circular dependency 😄